core/common/src/main/java/alluxio/underfs/UnderFileSystemFactoryRegistry.java

```diff
@@ -69,6 +77,8 @@
 public final class UnderFileSystemFactoryRegistry {
   private static final Logger LOG = LoggerFactory.getLogger(UnderFileSystemFactoryRegistry.class);
 
+  // Key: JAR name
+  private static final Set<String> LOADED_EXTENSION_KEYS = new HashSet<>();
```

Comment:
If we call it `LOADED_EXTENSION_JAR_NAMES`, we don't need the comment to explain that by "key" we mean "jar name"